### ISSUE TITLE
DV in regression results

### ISSUE THREAD

**Issue by App-State-Prof** [jasp-stats/jasp-desktop#2120]
It would be great for students if the linear regression test results would include the dependent variable for each test. They sometimes copy and save results, but forget to mark the DV. Thank you.

**Comment by EJWagenmakers** [comment]
Good idea; could we include this information in the table somewhere, @JohnnyDoorn?
